src/document/crdt/rga_tree_split.ts

```diff
@@ -485,7 +467,7 @@ export class RGATreeSplit<T extends RGATreeSplitValue> {
     editedAt: TimeTicket,
     value?: T,
     latestCreatedAtMapByActor?: Map<string, TimeTicket>,
-  ): [RGATreeSplitNodePos, Map<string, TimeTicket>, Array<TextChange>] {
+  ): [RGATreeSplitNodePos, Map<string, TimeTicket>, Array<ContentChange>] {
```

Comment:
we are also using `Content` as a type of `TextChange`, but I think it can be confusing since we decided to add `ContentChange` to remove dependency from Text.

I guess one of the solutions is using `Content` for low levels(common data structure) and `Value` for CRDT, JSON-Like levels.
(then TextChange types would be `Value`, `Style`, and `Selection`.)